rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, :person, :book
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    @book = book
+    book.rent_books(self)
```

Comment:
- [ ] This is supposed to be `the rental` adding itself to the `@book` the 'belongs-to' association
`book.rentals << self`